app/assets/javascripts/sign_up_async_request.js.erb

```diff
@@ -95,32 +86,52 @@ function validateEmailDomain(email){
 	var domain = email.split('@')
 	if(domain[1] != "schools.nyc.gov"){
 		$(".spinner").css("display","none");
-	    $('#check_email_div').css("display","none");
+		$('#check_email_div').css("display","none");
 		throw new InvalidEmailDomainError("Invalid e-mail domain")
 	}
 }
 
 
 window.ClientSideValidations.callbacks.element.fail = function(element, message, callback) {
 	callback();
+	$("#sign_up_button").css("background-color","grey");
 	$(element).css('background', '#EEB4B8');
 	$(element).css('border','0.0625rem #45403A solid');
 	var element_id = $(element).attr('id'); 
 	$($('label[for=' + element_id + ']')[1]).attr('id', element_id + "_error")
+	$(element).attr('validation-passed',false)
 	if(element[0].name == 'user[email]'){
 		$($("[for=user_email]")[0]).text('Enter a valid email address ending in "@schools.nyc.gov".')
 		$($('label[for=' + element_id + ']')[0]).attr('id', element_id + "_error")
 	}
 }
 
+
 window.ClientSideValidations.callbacks.element.pass = function(element, callback) {
 	$(element).css('background', '#E8E4E2');
 	$(element).css('border','1px solid #D0CDC8');
+	$(element).attr('validation-passed',true)
```

Comment:
A linter would recommend adding a space after the comma.

Reply:
Removed this line.